projects/uswds-formly/src/lib/wrappers/form-field.wrapper.ts

```diff
@@ -9,15 +9,20 @@ import { FieldWrapper } from '@ngx-formly/core';
 
 @Component({
   template: `
-   <div class="usa-form-group" [class.usa-form-group--error]="showError">
+    <div class="usa-form-group" [class.usa-form-group--error]="showError">
       <label
         class="usa-label"
         *ngIf="to.label && !to.hideLabel"
         [attr.for]="id"
       >
-       {{ to.label }}
+        {{ to.label }}
       </label>
-      <small *ngIf="to.description" class="form-text text-muted" [innerHTML]="to.description"></small>
+      <div
+        *ngIf="to.description"
+        class="usa-label--description text-muted"
```

Comment:
Do these 2 classes exist in uswds or are these sam-styles only classes?

Reply:
its for sam-styles

Reply:
We should not use sam-styles only classes within uswds projects. This project should work with just uswds. The request to update description might have been for sds repo and not this repo

Reply:
usa-hint class updated for the description